src/IotHub/IotHub/Az.IotHub.psd1

```diff
@@ -56,7 +56,8 @@ DotNetFrameworkVersion = '4.7.2'
 RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.7.0'; })
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = 'Microsoft.Azure.Management.IotHub.dll'
+RequiredAssemblies = 'Microsoft.Azure.Management.IotHub.dll', 'Microsoft.Azure.Devices.Shared.dll',
+					'Microsoft.Azure.Devices.dll', 'Microsoft.Azure.Amqp.dll'
```

Comment:
are 'Microsoft.Azure.Devices.Shared.dll' and 'Microsoft.Azure.Amqp.dll' still needed since you removed references from csproj?

Reply:
These are included in Microsoft.azure.devices.dll and giving an error in the build if removes.